Python/sysmodule.c

```diff
@@ -2123,12 +2123,19 @@ extern PyObject *_Py_GetObjects(PyObject *, PyObject *);
 sys._clear_type_cache
 
 Clear the internal type lookup cache.
+
+This function is deprecated and will be removed in future python versions. May we should specify 3.15?
 [clinic start generated code]*/
 
 static PyObject *
 sys__clear_type_cache_impl(PyObject *module)
-/*[clinic end generated code: output=20e48ca54a6f6971 input=127f3e04a8d9b555]*/
+/*[clinic end generated code: output=20e48ca54a6f6971 input=3997d1000692310e]*/
 {
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "sys._clear_type_cache() is deprecated "
+                     "and will be removed in future python versions", 1)) {
```

Comment:
Let's use the recommendation from the docs:
```suggestion
                     "sys._clear_type_cache() is deprecated and scheduled for removal in a "
                     "future version. Use the more general sys._clear_internal_caches() "
                     "function instead.", 1)) {
```